services/couchdb/Dockerfile

```diff
@@ -1 +1 @@
-FROM klaemo/couchdb-ssl:latest
+FROM klaemo/couchdb:latest
```

Comment:
Looks like the official image is younger than this pull request :smile: I felt safe using this one because the author is an official CouchDB contributor. I'll test the official one.